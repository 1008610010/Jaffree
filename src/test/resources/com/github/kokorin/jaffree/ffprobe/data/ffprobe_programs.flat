programs.program.0.program_id=1
programs.program.0.program_num=1
programs.program.0.nb_streams=2
programs.program.0.pmt_pid=4096
programs.program.0.pcr_pid=256
programs.program.0.start_pts=1433367
programs.program.0.start_time="1.433367"
programs.program.0.end_pts=52751300
programs.program.0.end_time="52.751300"
programs.program.0.tags.service_name="Ram"
programs.program.0.tags.service_provider="FFmpeg"
programs.program.0.streams.stream.0.index=0
programs.program.0.streams.stream.0.codec_name="mpeg2video"
programs.program.0.streams.stream.0.codec_long_name="MPEG-2 video"
programs.program.0.streams.stream.0.profile="Main"
programs.program.0.streams.stream.0.codec_type="video"
programs.program.0.streams.stream.0.codec_time_base="1001/30000"
programs.program.0.streams.stream.0.codec_tag_string="[2][0][0][0]"
programs.program.0.streams.stream.0.codec_tag="0x0002"
programs.program.0.streams.stream.0.width=720
programs.program.0.streams.stream.0.height=480
programs.program.0.streams.stream.0.coded_width=0
programs.program.0.streams.stream.0.coded_height=0
programs.program.0.streams.stream.0.has_b_frames=1
programs.program.0.streams.stream.0.sample_aspect_ratio="32:27"
programs.program.0.streams.stream.0.display_aspect_ratio="16:9"
programs.program.0.streams.stream.0.pix_fmt="yuv420p"
programs.program.0.streams.stream.0.level=8
programs.program.0.streams.stream.0.color_range="tv"
programs.program.0.streams.stream.0.color_space="unknown"
programs.program.0.streams.stream.0.color_transfer="unknown"
programs.program.0.streams.stream.0.color_primaries="unknown"
programs.program.0.streams.stream.0.chroma_location="left"
programs.program.0.streams.stream.0.field_order="progressive"
programs.program.0.streams.stream.0.timecode="N/A"
programs.program.0.streams.stream.0.refs=1
programs.program.0.streams.stream.0.id="0x100"
programs.program.0.streams.stream.0.r_frame_rate="30000/1001"
programs.program.0.streams.stream.0.avg_frame_rate="30000/1001"
programs.program.0.streams.stream.0.time_base="1/90000"
programs.program.0.streams.stream.0.start_pts=129003
programs.program.0.streams.stream.0.start_time="1.433367"
programs.program.0.streams.stream.0.duration_ts=4618614
programs.program.0.streams.stream.0.duration="51.317933"
programs.program.0.streams.stream.0.bit_rate="N/A"
programs.program.0.streams.stream.0.max_bit_rate="N/A"
programs.program.0.streams.stream.0.bits_per_raw_sample="N/A"
programs.program.0.streams.stream.0.nb_frames="N/A"
programs.program.0.streams.stream.0.nb_read_frames="N/A"
programs.program.0.streams.stream.0.nb_read_packets="N/A"
programs.program.0.streams.stream.0.disposition.default=0
programs.program.0.streams.stream.0.disposition.dub=0
programs.program.0.streams.stream.0.disposition.original=0
programs.program.0.streams.stream.0.disposition.comment=0
programs.program.0.streams.stream.0.disposition.lyrics=0
programs.program.0.streams.stream.0.disposition.karaoke=0
programs.program.0.streams.stream.0.disposition.forced=0
programs.program.0.streams.stream.0.disposition.hearing_impaired=0
programs.program.0.streams.stream.0.disposition.visual_impaired=0
programs.program.0.streams.stream.0.disposition.clean_effects=0
programs.program.0.streams.stream.0.disposition.attached_pic=0
programs.program.0.streams.stream.0.disposition.timed_thumbnails=0
programs.program.0.streams.stream.1.index=1
programs.program.0.streams.stream.1.codec_name="mp2"
programs.program.0.streams.stream.1.codec_long_name="MP2 (MPEG audio layer 2)"
programs.program.0.streams.stream.1.profile="unknown"
programs.program.0.streams.stream.1.codec_type="audio"
programs.program.0.streams.stream.1.codec_time_base="1/48000"
programs.program.0.streams.stream.1.codec_tag_string="[3][0][0][0]"
programs.program.0.streams.stream.1.codec_tag="0x0003"
programs.program.0.streams.stream.1.sample_fmt="s16p"
programs.program.0.streams.stream.1.sample_rate="48000"
programs.program.0.streams.stream.1.channels=2
programs.program.0.streams.stream.1.channel_layout="stereo"
programs.program.0.streams.stream.1.bits_per_sample=0
programs.program.0.streams.stream.1.id="0x101"
programs.program.0.streams.stream.1.r_frame_rate="0/0"
programs.program.0.streams.stream.1.avg_frame_rate="0/0"
programs.program.0.streams.stream.1.time_base="1/90000"
programs.program.0.streams.stream.1.start_pts=131647
programs.program.0.streams.stream.1.start_time="1.462744"
programs.program.0.streams.stream.1.duration_ts=4608720
programs.program.0.streams.stream.1.duration="51.208000"
programs.program.0.streams.stream.1.bit_rate="384000"
programs.program.0.streams.stream.1.max_bit_rate="N/A"
programs.program.0.streams.stream.1.bits_per_raw_sample="N/A"
programs.program.0.streams.stream.1.nb_frames="N/A"
programs.program.0.streams.stream.1.nb_read_frames="N/A"
programs.program.0.streams.stream.1.nb_read_packets="N/A"
programs.program.0.streams.stream.1.disposition.default=0
programs.program.0.streams.stream.1.disposition.dub=0
programs.program.0.streams.stream.1.disposition.original=0
programs.program.0.streams.stream.1.disposition.comment=0
programs.program.0.streams.stream.1.disposition.lyrics=0
programs.program.0.streams.stream.1.disposition.karaoke=0
programs.program.0.streams.stream.1.disposition.forced=0
programs.program.0.streams.stream.1.disposition.hearing_impaired=0
programs.program.0.streams.stream.1.disposition.visual_impaired=0
programs.program.0.streams.stream.1.disposition.clean_effects=0
programs.program.0.streams.stream.1.disposition.attached_pic=0
programs.program.0.streams.stream.1.disposition.timed_thumbnails=0
programs.program.1.program_id=2
programs.program.1.program_num=2
programs.program.1.nb_streams=2
programs.program.1.pmt_pid=4097
programs.program.1.pcr_pid=258
programs.program.1.start_pts=1433367
programs.program.1.start_time="1.433367"
programs.program.1.end_pts=52751300
programs.program.1.end_time="52.751300"
programs.program.1.tags.service_name="Linc"
programs.program.1.tags.service_provider="FFmpeg"
programs.program.1.streams.stream.0.index=2
programs.program.1.streams.stream.0.codec_name="mpeg2video"
programs.program.1.streams.stream.0.codec_long_name="MPEG-2 video"
programs.program.1.streams.stream.0.profile="Main"
programs.program.1.streams.stream.0.codec_type="video"
programs.program.1.streams.stream.0.codec_time_base="1001/30000"
programs.program.1.streams.stream.0.codec_tag_string="[2][0][0][0]"
programs.program.1.streams.stream.0.codec_tag="0x0002"
programs.program.1.streams.stream.0.width=720
programs.program.1.streams.stream.0.height=480
programs.program.1.streams.stream.0.coded_width=0
programs.program.1.streams.stream.0.coded_height=0
programs.program.1.streams.stream.0.has_b_frames=1
programs.program.1.streams.stream.0.sample_aspect_ratio="32:27"
programs.program.1.streams.stream.0.display_aspect_ratio="16:9"
programs.program.1.streams.stream.0.pix_fmt="yuv420p"
programs.program.1.streams.stream.0.level=8
programs.program.1.streams.stream.0.color_range="tv"
programs.program.1.streams.stream.0.color_space="unknown"
programs.program.1.streams.stream.0.color_transfer="unknown"
programs.program.1.streams.stream.0.color_primaries="unknown"
programs.program.1.streams.stream.0.chroma_location="left"
programs.program.1.streams.stream.0.field_order="progressive"
programs.program.1.streams.stream.0.timecode="N/A"
programs.program.1.streams.stream.0.refs=1
programs.program.1.streams.stream.0.id="0x102"
programs.program.1.streams.stream.0.r_frame_rate="30000/1001"
programs.program.1.streams.stream.0.avg_frame_rate="30000/1001"
programs.program.1.streams.stream.0.time_base="1/90000"
programs.program.1.streams.stream.0.start_pts=129003
programs.program.1.streams.stream.0.start_time="1.433367"
programs.program.1.streams.stream.0.duration_ts=4618614
programs.program.1.streams.stream.0.duration="51.317933"
programs.program.1.streams.stream.0.bit_rate="N/A"
programs.program.1.streams.stream.0.max_bit_rate="N/A"
programs.program.1.streams.stream.0.bits_per_raw_sample="N/A"
programs.program.1.streams.stream.0.nb_frames="N/A"
programs.program.1.streams.stream.0.nb_read_frames="N/A"
programs.program.1.streams.stream.0.nb_read_packets="N/A"
programs.program.1.streams.stream.0.disposition.default=0
programs.program.1.streams.stream.0.disposition.dub=0
programs.program.1.streams.stream.0.disposition.original=0
programs.program.1.streams.stream.0.disposition.comment=0
programs.program.1.streams.stream.0.disposition.lyrics=0
programs.program.1.streams.stream.0.disposition.karaoke=0
programs.program.1.streams.stream.0.disposition.forced=0
programs.program.1.streams.stream.0.disposition.hearing_impaired=0
programs.program.1.streams.stream.0.disposition.visual_impaired=0
programs.program.1.streams.stream.0.disposition.clean_effects=0
programs.program.1.streams.stream.0.disposition.attached_pic=0
programs.program.1.streams.stream.0.disposition.timed_thumbnails=0
programs.program.1.streams.stream.1.index=3
programs.program.1.streams.stream.1.codec_name="mp2"
programs.program.1.streams.stream.1.codec_long_name="MP2 (MPEG audio layer 2)"
programs.program.1.streams.stream.1.profile="unknown"
programs.program.1.streams.stream.1.codec_type="audio"
programs.program.1.streams.stream.1.codec_time_base="1/48000"
programs.program.1.streams.stream.1.codec_tag_string="[3][0][0][0]"
programs.program.1.streams.stream.1.codec_tag="0x0003"
programs.program.1.streams.stream.1.sample_fmt="s16p"
programs.program.1.streams.stream.1.sample_rate="48000"
programs.program.1.streams.stream.1.channels=2
programs.program.1.streams.stream.1.channel_layout="stereo"
programs.program.1.streams.stream.1.bits_per_sample=0
programs.program.1.streams.stream.1.id="0x103"
programs.program.1.streams.stream.1.r_frame_rate="0/0"
programs.program.1.streams.stream.1.avg_frame_rate="0/0"
programs.program.1.streams.stream.1.time_base="1/90000"
programs.program.1.streams.stream.1.start_pts=131647
programs.program.1.streams.stream.1.start_time="1.462744"
programs.program.1.streams.stream.1.duration_ts=4608720
programs.program.1.streams.stream.1.duration="51.208000"
programs.program.1.streams.stream.1.bit_rate="384000"
programs.program.1.streams.stream.1.max_bit_rate="N/A"
programs.program.1.streams.stream.1.bits_per_raw_sample="N/A"
programs.program.1.streams.stream.1.nb_frames="N/A"
programs.program.1.streams.stream.1.nb_read_frames="N/A"
programs.program.1.streams.stream.1.nb_read_packets="N/A"
programs.program.1.streams.stream.1.disposition.default=0
programs.program.1.streams.stream.1.disposition.dub=0
programs.program.1.streams.stream.1.disposition.original=0
programs.program.1.streams.stream.1.disposition.comment=0
programs.program.1.streams.stream.1.disposition.lyrics=0
programs.program.1.streams.stream.1.disposition.karaoke=0
programs.program.1.streams.stream.1.disposition.forced=0
programs.program.1.streams.stream.1.disposition.hearing_impaired=0
programs.program.1.streams.stream.1.disposition.visual_impaired=0
programs.program.1.streams.stream.1.disposition.clean_effects=0
programs.program.1.streams.stream.1.disposition.attached_pic=0
programs.program.1.streams.stream.1.disposition.timed_thumbnails=0
programs.program.2.program_id=3
programs.program.2.program_num=3
programs.program.2.nb_streams=2
programs.program.2.pmt_pid=4098
programs.program.2.pcr_pid=260
programs.program.2.start_pts=1433367
programs.program.2.start_time="1.433367"
programs.program.2.end_pts=52751300
programs.program.2.end_time="52.751300"
programs.program.2.tags.service_name="Bush"
programs.program.2.tags.service_provider="FFmpeg"
programs.program.2.streams.stream.0.index=4
programs.program.2.streams.stream.0.codec_name="mpeg2video"
programs.program.2.streams.stream.0.codec_long_name="MPEG-2 video"
programs.program.2.streams.stream.0.profile="Main"
programs.program.2.streams.stream.0.codec_type="video"
programs.program.2.streams.stream.0.codec_time_base="1001/30000"
programs.program.2.streams.stream.0.codec_tag_string="[2][0][0][0]"
programs.program.2.streams.stream.0.codec_tag="0x0002"
programs.program.2.streams.stream.0.width=720
programs.program.2.streams.stream.0.height=480
programs.program.2.streams.stream.0.coded_width=0
programs.program.2.streams.stream.0.coded_height=0
programs.program.2.streams.stream.0.has_b_frames=1
programs.program.2.streams.stream.0.sample_aspect_ratio="32:27"
programs.program.2.streams.stream.0.display_aspect_ratio="16:9"
programs.program.2.streams.stream.0.pix_fmt="yuv420p"
programs.program.2.streams.stream.0.level=8
programs.program.2.streams.stream.0.color_range="tv"
programs.program.2.streams.stream.0.color_space="unknown"
programs.program.2.streams.stream.0.color_transfer="unknown"
programs.program.2.streams.stream.0.color_primaries="unknown"
programs.program.2.streams.stream.0.chroma_location="left"
programs.program.2.streams.stream.0.field_order="progressive"
programs.program.2.streams.stream.0.timecode="N/A"
programs.program.2.streams.stream.0.refs=1
programs.program.2.streams.stream.0.id="0x104"
programs.program.2.streams.stream.0.r_frame_rate="30000/1001"
programs.program.2.streams.stream.0.avg_frame_rate="30000/1001"
programs.program.2.streams.stream.0.time_base="1/90000"
programs.program.2.streams.stream.0.start_pts=129003
programs.program.2.streams.stream.0.start_time="1.433367"
programs.program.2.streams.stream.0.duration_ts=4618614
programs.program.2.streams.stream.0.duration="51.317933"
programs.program.2.streams.stream.0.bit_rate="N/A"
programs.program.2.streams.stream.0.max_bit_rate="N/A"
programs.program.2.streams.stream.0.bits_per_raw_sample="N/A"
programs.program.2.streams.stream.0.nb_frames="N/A"
programs.program.2.streams.stream.0.nb_read_frames="N/A"
programs.program.2.streams.stream.0.nb_read_packets="N/A"
programs.program.2.streams.stream.0.disposition.default=0
programs.program.2.streams.stream.0.disposition.dub=0
programs.program.2.streams.stream.0.disposition.original=0
programs.program.2.streams.stream.0.disposition.comment=0
programs.program.2.streams.stream.0.disposition.lyrics=0
programs.program.2.streams.stream.0.disposition.karaoke=0
programs.program.2.streams.stream.0.disposition.forced=0
programs.program.2.streams.stream.0.disposition.hearing_impaired=0
programs.program.2.streams.stream.0.disposition.visual_impaired=0
programs.program.2.streams.stream.0.disposition.clean_effects=0
programs.program.2.streams.stream.0.disposition.attached_pic=0
programs.program.2.streams.stream.0.disposition.timed_thumbnails=0
programs.program.2.streams.stream.1.index=5
programs.program.2.streams.stream.1.codec_name="mp2"
programs.program.2.streams.stream.1.codec_long_name="MP2 (MPEG audio layer 2)"
programs.program.2.streams.stream.1.profile="unknown"
programs.program.2.streams.stream.1.codec_type="audio"
programs.program.2.streams.stream.1.codec_time_base="1/48000"
programs.program.2.streams.stream.1.codec_tag_string="[3][0][0][0]"
programs.program.2.streams.stream.1.codec_tag="0x0003"
programs.program.2.streams.stream.1.sample_fmt="s16p"
programs.program.2.streams.stream.1.sample_rate="48000"
programs.program.2.streams.stream.1.channels=2
programs.program.2.streams.stream.1.channel_layout="stereo"
programs.program.2.streams.stream.1.bits_per_sample=0
programs.program.2.streams.stream.1.id="0x105"
programs.program.2.streams.stream.1.r_frame_rate="0/0"
programs.program.2.streams.stream.1.avg_frame_rate="0/0"
programs.program.2.streams.stream.1.time_base="1/90000"
programs.program.2.streams.stream.1.start_pts=131647
programs.program.2.streams.stream.1.start_time="1.462744"
programs.program.2.streams.stream.1.duration_ts=4608720
programs.program.2.streams.stream.1.duration="51.208000"
programs.program.2.streams.stream.1.bit_rate="384000"
programs.program.2.streams.stream.1.max_bit_rate="N/A"
programs.program.2.streams.stream.1.bits_per_raw_sample="N/A"
programs.program.2.streams.stream.1.nb_frames="N/A"
programs.program.2.streams.stream.1.nb_read_frames="N/A"
programs.program.2.streams.stream.1.nb_read_packets="N/A"
programs.program.2.streams.stream.1.disposition.default=0
programs.program.2.streams.stream.1.disposition.dub=0
programs.program.2.streams.stream.1.disposition.original=0
programs.program.2.streams.stream.1.disposition.comment=0
programs.program.2.streams.stream.1.disposition.lyrics=0
programs.program.2.streams.stream.1.disposition.karaoke=0
programs.program.2.streams.stream.1.disposition.forced=0
programs.program.2.streams.stream.1.disposition.hearing_impaired=0
programs.program.2.streams.stream.1.disposition.visual_impaired=0
programs.program.2.streams.stream.1.disposition.clean_effects=0
programs.program.2.streams.stream.1.disposition.attached_pic=0
programs.program.2.streams.stream.1.disposition.timed_thumbnails=0
