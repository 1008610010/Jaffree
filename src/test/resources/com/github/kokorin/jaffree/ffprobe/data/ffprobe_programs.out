[PROGRAM]
program_id=1
program_num=1
nb_streams=2
pmt_pid=4096
pcr_pid=256
start_pts=1433367
start_time=1.433367
end_pts=52751300
end_time=52.751300
TAG:service_name=Ram
TAG:service_provider=FFmpeg
[STREAM]
index=0
codec_name=mpeg2video
codec_long_name=MPEG-2 video
profile=Main
codec_type=video
codec_time_base=1001/30000
codec_tag_string=[2][0][0][0]
codec_tag=0x0002
width=720
height=480
coded_width=0
coded_height=0
has_b_frames=1
sample_aspect_ratio=32:27
display_aspect_ratio=16:9
pix_fmt=yuv420p
level=8
color_range=tv
color_space=unknown
color_transfer=unknown
color_primaries=unknown
chroma_location=left
field_order=progressive
timecode=N/A
refs=1
id=0x100
r_frame_rate=30000/1001
avg_frame_rate=30000/1001
time_base=1/90000
start_pts=129003
start_time=1.433367
duration_ts=4618614
duration=51.317933
bit_rate=N/A
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
[/STREAM]
[STREAM]
index=1
codec_name=mp2
codec_long_name=MP2 (MPEG audio layer 2)
profile=unknown
codec_type=audio
codec_time_base=1/48000
codec_tag_string=[3][0][0][0]
codec_tag=0x0003
sample_fmt=s16p
sample_rate=48000
channels=2
channel_layout=stereo
bits_per_sample=0
id=0x101
r_frame_rate=0/0
avg_frame_rate=0/0
time_base=1/90000
start_pts=131647
start_time=1.462744
duration_ts=4608720
duration=51.208000
bit_rate=384000
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
[/STREAM]
[/PROGRAM]
[PROGRAM]
program_id=2
program_num=2
nb_streams=2
pmt_pid=4097
pcr_pid=258
start_pts=1433367
start_time=1.433367
end_pts=52751300
end_time=52.751300
TAG:service_name=Linc
TAG:service_provider=FFmpeg
[STREAM]
index=2
codec_name=mpeg2video
codec_long_name=MPEG-2 video
profile=Main
codec_type=video
codec_time_base=1001/30000
codec_tag_string=[2][0][0][0]
codec_tag=0x0002
width=720
height=480
coded_width=0
coded_height=0
has_b_frames=1
sample_aspect_ratio=32:27
display_aspect_ratio=16:9
pix_fmt=yuv420p
level=8
color_range=tv
color_space=unknown
color_transfer=unknown
color_primaries=unknown
chroma_location=left
field_order=progressive
timecode=N/A
refs=1
id=0x102
r_frame_rate=30000/1001
avg_frame_rate=30000/1001
time_base=1/90000
start_pts=129003
start_time=1.433367
duration_ts=4618614
duration=51.317933
bit_rate=N/A
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
[/STREAM]
[STREAM]
index=3
codec_name=mp2
codec_long_name=MP2 (MPEG audio layer 2)
profile=unknown
codec_type=audio
codec_time_base=1/48000
codec_tag_string=[3][0][0][0]
codec_tag=0x0003
sample_fmt=s16p
sample_rate=48000
channels=2
channel_layout=stereo
bits_per_sample=0
id=0x103
r_frame_rate=0/0
avg_frame_rate=0/0
time_base=1/90000
start_pts=131647
start_time=1.462744
duration_ts=4608720
duration=51.208000
bit_rate=384000
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
[/STREAM]
[/PROGRAM]
[PROGRAM]
program_id=3
program_num=3
nb_streams=2
pmt_pid=4098
pcr_pid=260
start_pts=1433367
start_time=1.433367
end_pts=52751300
end_time=52.751300
TAG:service_name=Bush
TAG:service_provider=FFmpeg
[STREAM]
index=4
codec_name=mpeg2video
codec_long_name=MPEG-2 video
profile=Main
codec_type=video
codec_time_base=1001/30000
codec_tag_string=[2][0][0][0]
codec_tag=0x0002
width=720
height=480
coded_width=0
coded_height=0
has_b_frames=1
sample_aspect_ratio=32:27
display_aspect_ratio=16:9
pix_fmt=yuv420p
level=8
color_range=tv
color_space=unknown
color_transfer=unknown
color_primaries=unknown
chroma_location=left
field_order=progressive
timecode=N/A
refs=1
id=0x104
r_frame_rate=30000/1001
avg_frame_rate=30000/1001
time_base=1/90000
start_pts=129003
start_time=1.433367
duration_ts=4618614
duration=51.317933
bit_rate=N/A
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
[/STREAM]
[STREAM]
index=5
codec_name=mp2
codec_long_name=MP2 (MPEG audio layer 2)
profile=unknown
codec_type=audio
codec_time_base=1/48000
codec_tag_string=[3][0][0][0]
codec_tag=0x0003
sample_fmt=s16p
sample_rate=48000
channels=2
channel_layout=stereo
bits_per_sample=0
id=0x105
r_frame_rate=0/0
avg_frame_rate=0/0
time_base=1/90000
start_pts=131647
start_time=1.462744
duration_ts=4608720
duration=51.208000
bit_rate=384000
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
[/STREAM]
[/PROGRAM]
